src/docs/product/issues/issue-details/performance-issues/consecutive-db-queries/index.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: "Consecutive DB Queries"
+sidebar_order: 30
+redirect_from:
+  - /product/issues/performance-issues/consecutive-db-queries/
+description: "Learn more about Consecutive DB Queries and how to diagnose and fix them."
+---
+
+<Include name="early-adopter-note.mdx" />
+
+Consecutive DB Queries are a sequence of database spans where one or more have been identified as parallelizable, or in other words, spans that may be shifted to the start of the sequence. This often occurs when a db query performs no filtering on the data, for example a query without a WHERE clause. To learn more about how to fix consecutive DB queries, check out these resources:
+
+## Detection Criteria
+
+The detector for this performance issuee looks for a set of sequential, non-overlapping database spans (consecutive spans). Within this set of spans, it proceeds to look for queries that are parallelziable, which occurs when all following criteria is true about the query.
```

Comment:
```suggestion
The detector for this performance issue looks for a set of sequential, non-overlapping database spans. Within this set of spans, it proceeds to look for queries that are parallelizable, which occurs when all following criteria is true about the query:
```

some grammar and spelling corrections + we can remove `(consecutive spans)` since it's redundant